runtime-modules/content/src/lib.rs

```diff
@@ -2590,6 +2730,28 @@ impl<T: Trait> Module<T> {
             video_id, member_id,
         ))
     }
+
+    fn ensure_assets_to_remove_are_part_of_assets_set(
+        assets_to_remove: &BTreeSet<DataObjectId<T>>,
+        assets_set: &BTreeSet<DataObjectId<T>>,
+    ) -> DispatchResult {
+        ensure!(
+            assets_to_remove.iter().all(|id| assets_set.contains(id)),
```

Comment:
`assets_to_remove.is_subset(assets_set)`

Reply:
Done in https://github.com/Joystream/joystream/pull/3454/commits/d96affab416d06332652e74dcb0e3b914443c1f6